src/TaskBoard.js

```diff
@@ -5,10 +5,13 @@ import { useState } from "react"
 
 export default function TaskBoard(){
   const [task, setTask] = useState([]);
+  const [ taskInput, setTaskInupt] = useState('');
+  const [ asignee, setAsignee] = useState('')
   
   
   var taskID = task.length;
```

Comment:
@Tom-Sarfo we had a conversation about declaring variables with `var`

Reply:
Yeah I remember